src/I18n/MessagesFileLoader.php

```diff
@@ -160,7 +160,7 @@ public function translationsFolders(): array
 
         $searchPaths = [];
 
-        $localePaths = App::path('Locale');
+        $localePaths = App::path('locales');
```

Comment:
Is this something we should call out in the migration guide? I don't think rector can change argument values.

Reply:
Yes it should be mentioned in migration guide, I'll make a PR.

Reply:
Another option would be to continue handling the old names but with deprecation warnings.

Reply:
Removing the special handing of `Template` and `Locale` types would allow people to use them as namespaces if needed, so I would rather not keep the old names.